data/json/recipes/recipe_food.json

```diff
@@ -11524,6 +11708,98 @@
       [ "water_clean", 2 ]
     ]
   ]
+},{
+  "type" : "recipe",
+   "result": "sandwich_v",
+   "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_MEAT",
+   "skill_used": "cooking",
+   "difficulty": 0,
+   "time": 3000,
+   "reversible": true,
+   "autolearn": true,
+   "qualities" : [
+     {"id":"CUT","level":1,"amount":1}
+  ],
+  "components": [
+    [
+      [ "flatbread", 2 ],
+      [ "bread", 2 ],
+      [ "cornbread", 2 ],
+      [ "wastebread", 2 ]
+    ],
+    [
+      [ "veggy_cooked", 1 ],
+      [ "veggy_wild_cooked", 1 ],
+      [ "veggy_vac", 1 ],
+      [ "celery", 1 ],
+      [ "irradiated_celery", 1 ],
+      [ "cabbage", 1 ],
+      [ "irradiated_cabbage", 1 ],
+      [ "sauerkraut", 1 ],
+      [ "tomato", 1 ],
+      [ "can_tomato", 1 ],
+      [ "irradiated_pumpkin", 1 ],
+      [ "onion", 1 ],
+      [ "irradiated_onion", 1 ],
+      [ "broccoli", 1 ],
+      [ "irradiated_tomato", 1 ],
+      [ "irradiated_broccoli", 1 ],
+      [ "rehydrated_veggy", 1 ],
+      [ "morel_cooked", 1 ],
+      [ "mushroom_cooked", 1 ],
+      [ "sauerkraut_onions", 1 ],
+      [ "pickle", 1 ],
+      [ "veggy_canned", 1 ],
+      [ "beans_cooked", 1 ],
+      [ "irradiated_carrot", 1 ],
+      [ "irradiated_zucchini", 1 ],
+      [ "irradiated_lettuce", 1 ],
+      [ "carrot", 1 ],
+      [ "zucchini", 1 ],
+      [ "lettuce", 1 ],
+      [ "pine_nuts", 1 ],
+      [ "morel_fried", 1 ],
+      [ "veggy_aspic", 1 ],
+      [ "dahlia_baked", 1 ],
+      [ "irradiated_cucumber", 1 ],
+      [ "cucumber", 1 ],
+      [ "veggy_pickled", 1 ],
+      [ "cooked_cattail_stalk", 1 ],
+      [ "cooked_pumpkin", 1 ],
+      [ "potato_baked", 1 ],
+      [ "potato_irradiated", 1 ],
+      [ "can_tomato", 1 ],
+      [ "dandelion_cooked", 1 ]
+    ]
+  ]
+},
+{
+   "type" : "recipe",
+   "result": "sandwich_cucumber",
+   "category": "CC_FOOD",
+   "subcategory": "CSC_FOOD_SNACK",
+   "skill_used": "cooking",
+   "difficulty": 0,
+   "time": 1500,
+   "reversible": false,
+   "autolearn": true,
+   "qualities" : [
+     {"id":"CUT","level":1,"amount":1}
+   ],
+  "components": [
+    [
+      [ "flatbread", 2 ],
+      [ "bread", 2 ]
+    ],
+    [
+      [ "cucumber", 1 ],
+      [ "irradiated_cucumber", 1 ]
```

Comment:
Should probably allow pickles too.
Regular cucumbers will run out really quickly, making the recipe hard to actually ever use.
Allowing more bread types would also help.